Mathlib/MeasureTheory/Function/L1Space.lean

```diff
@@ -443,7 +443,7 @@ lemma Integrable.of_finite [Finite α] [MeasurableSingletonClass α] [IsFiniteMe
 
 /-- This lemma is a special case of `Integrable.of_finite`. -/
 -- Eternal deprecation for discoverability, don't remove
-@[deprecated Integrable.of_finite, nolint deprecatedNoSince]
+@[deprecated Integrable.of_finite (since := "2024-10-05"), nolint deprecatedNoSince]
```

Comment:
I guess the `nolint` doesn't do anything here? Is it used somewhere else in the library?

Reply:
I guess no `since` is intentional, as it never expires and hence is searchable by e.g. Loogle.

Reply:
I think we'll just remove the `deprecatedNoSince` linter shortly.